stable/prometheus-operator/values.yaml

```diff
@@ -590,6 +590,13 @@ kubeStateMetrics:
     ##
     interval: ""
 
+    ## 	metric relabel configs to apply to samples before ingestion.
+    ##
+    metricRelabelings: []
```

Comment:
Could you also add this to the `ci/` folder please?

Reply:
done thanks :)